sky/shell/platform_view.h

```diff
@@ -77,11 +77,14 @@ class PlatformView {
 
   std::unique_ptr<Rasterizer> rasterizer_;
   std::unique_ptr<Engine> engine_;
+  SkyEnginePtr engine_ptr_;
```

Comment:
We have `std::unique_ptr<Engine> engine_` here.  If you're on the proper thread (the UI thread), you can just call functions on that directly.  You don't have to go through the mojo pipe.